# @Exception

{
  let x = 1
  let x = 2
}
